docs/src/specification/sync_leave.rst

```diff
@@ -0,0 +1,15 @@
+Sync Leave
+==========
+
+The sync leave protocol is used to command the repo to leave the sync group. This command also removes any information
+about the sync group from repo storage.
+
+1. The repo subscribes to the topic ``/<repo_name>/sync/leave``.
+
+2. The client publishes a message to the topic ``/<repo_name>/sync/leave``. The message payload is of type
+   ``RepoCommandParam``, containing one or more ``SyncParam`` with the following field:
+
+  * ``sync_prefix``: The name prefix of the sync group to leave.
+
+3. The repo leaves the sync group, removing sync information from disk. The repo no longer listens to the originally
```

Comment:
I believe we also need to clarify sync leave does not delete any already stored Data packets.